keras_cv/losses/ciou_loss_test.py

```diff
@@ -65,21 +66,38 @@ def test_output_shape_relative_formats(self):
         self.assertAllEqual(ciou_loss(y_true, y_pred).shape, ())
 
     def test_output_value(self):
-        y_true = [
-            [0, 0, 1, 1],
-            [0, 0, 2, 3],
-            [4, 5, 3, 6],
-            [2, 2, 3, 3],
-        ]
-
-        y_pred = [
-            [0, 0, 5, 6],
-            [0, 0, 7, 3],
-            [4, 5, 5, 6],
-            [2, 1, 3, 3],
-        ]
-
-        ciou_loss = CIoULoss(bounding_box_format="xyxy")
-
-        # expected value for these values is 1.0320253
-        self.assertAllClose(ciou_loss(y_true, y_pred), 1.0320253)
+        def test_loss_format(bounding_box_format):
+            y_true = [
+                [0, 0, 1, 1],
+                [0, 0, 2, 3],
+                [4, 5, 3, 6],
+                [2, 2, 3, 3],
+            ]
+
+            y_pred = [
+                [0, 0, 5, 6],
+                [0, 0, 7, 3],
+                [4, 5, 5, 6],
+                [2, 1, 3, 3],
+            ]
+
+            if bounding_box_format == "xyxy":
+                ciou_loss = CIoULoss(bounding_box_format="xyxy")
+                # expected value is 1.032025
+                expected_loss = 1.032025
```

Comment:
For these two, we can just specify `expected_loss = 1.03202` for both versions, as the difference is just a tiny numerical error.

If the test fails for one version, we can specify `atol=0.0001` in the `assertAllClose`